source/_integrations/zha.markdown

```diff
@@ -725,10 +725,22 @@ services:
     network_mode: host
 ```
 
-### EZSP error and other log messages
+### Common error codes in ZHA logs and other log messages
 
-#### NCP entered failed state
+#### EZSP error - NCP entered failed state
 
-When you see `NCP entered failed state. Requesting APP controller restart` in logs during normal operation, it indicates a drop in communication between ZHA and the serial interface of the Silabs EmberZNet Zigbee Coordinator.
+When you see `NCP entered failed state. Requesting APP controller restart` in ZHA logs during normal operation, that indicates was or is a drop in communication between ZHA and the serial interface of the Silabs EmberZNet Zigbee Coordinator (EZSP).
```

Comment:
Wording such as "that indicates was or is a drop in" is extremely unclear and the time period for when this happened is not relevant. Let's remove unnecessary text wherever possible:

```suggestion
When you see `NCP entered failed state. Requesting APP controller restart` in ZHA logs during normal operation,  there was drop in communication between ZHA and the coordinator.
```